README.md

```diff
@@ -218,5 +202,26 @@ The (proof of concept) resync system, on the top right portion of the patch, inc
 
 Also don't forget to checkout our [wiki](https://github.com/omarcostahamido/PVM/wiki)! It contains instructions on various topics like [_setting up remote access to the Raspberry Pi_](https://github.com/omarcostahamido/PVM/wiki/How-to-connect-to-a-Raspberry-Pi-remotely) and how to run the examples.
 
+## Development
+
+### Logs
+
+All logs for each time each RPI is stored in the `log` folder in the current directory. If it doesn't exist yet, a new directory named `log` would be created.
+
+Name convention for each file is `{:%Y-%m-%d %H:%M:%S}.log`
+
+All output from the console is synchronized to the file in real-time.
+
+```bash
+2022-10-31 13:34:21;INFO;Received command: file
+2022-10-31 13:34:21;INFO;Received value: jellyfish720p.mp4
+2022-10-31 13:34:21;INFO;File set: /home/pi/Videos/jellyfish720p.mp4
+```
+
+If you close the program and then reopen it, a new log file will be created.
+
+## Run the test
+
+You could write tests for your combination of commands in the `test.py` file. Simply run `python3 test.py` from the console will run it. When the test script is started, it will first **kill any existing `pvm.py` processes**, and then spawn a new one to test the commands, and end it again after test is finished.
```

Comment:
`Simply run ``python3 test.py`` from the console will run it.` is confusing
maybe `Simply run ``python3 test.py`` on the console to perform the tests.`